css/styles.css

```diff
@@ -188,3 +201,199 @@ Headline
   font-size: 2rem;
   padding: 0.4rem;
 }
+
+/*
+=====================================
+My works section
+=====================================
+*/
+
+.works {
+  padding: 0 var(--horizontal-spacing);
+}
+
+.works__title,
+.aboutme__title {
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.works__title-underline {
+  border: 3px solid #36b37e;
+  background-color: #36b37e;
+  border-radius: 4px;
+  width: 50px;
+  margin-inline: auto;
+  margin-bottom: 3.5rem;
+}
+
+.works__cards {
+  display: grid;
+  gap: 1.5rem;
+  margin-bottom: 1.5rem;
+}
+
+.works__card {
+  border: 3px solid #d4dcd8;
+  border-radius: var(--border-radius);
+  overflow: hidden;
+}
```

Comment:
- [ ] Kindly make sure the contents of each individual project are aligned using flexbox in order to meet the project requirements. :wink: